securedrop/source_templates/session_timeout.html

```diff
@@ -0,0 +1,5 @@
+<div class="icon">
+  <img src="{{ url_for('static', filename='i/hand_with_fingerprint.png') }}">
+</div>
+<div class="message"><strong>{{ gettext('Important!') }}</strong><br>
+<p>{{ gettext('Your session timed out due to inactivity. Please login again if you want to continue using SecureDrop, or select "New Identity" from the green onion button in the Tor browser to clear all history of your SecureDrop usage from this device.') }}</p></div>
```

Comment:
Since the session cookies are non-persistent, this should say "Or if you're not using Tor Browser, please close all windows and tabs on your current browser" or whatever else. This catches the edge cases of people who aren't using TB.